src/mail/MailEditorN.js

```diff
@@ -449,7 +449,10 @@ function createMailEditorDialog(model: SendMailModel, blockExternalContent: bool
 		middle: () => conversationTypeString(model.getConversationType()),
 		create: () => {
 			windowCloseUnsubscribe = windowFacade.addWindowCloseListener(() => {
-				closeButtonAttrs.click(newMouseEvent(), domCloseButton)
+				// Simulate a button click only if we're on not on browser, since on a browser, we can't override it closing, only ask if the user is sure they want to close
+				if (!isBrowser()) {
+					closeButtonAttrs.click(newMouseEvent(), domCloseButton)
```

Comment:
I don't know if we need to do anything outside of browser, there are no tabs outside of browser anyway

Reply:
It looks like the event fires when you click the close button on the desktop client's window, too.

Currently, if you close the window on the desktop client, it will show a little drop down next to the Close button as if you clicked that, giving you the option to save a draft. Or, if you didn't type anything, it simply closes the mail editor.

Reply:
Didn't remember about desktop, maybe it's worth mentioning it